bentoml/repository/__init__.py

```diff
@@ -137,6 +137,10 @@ def __init__(self, base_url, s3_endpoint_url=None):
         self.base_path = parse_result.path.lstrip('/')
 
         s3_client_args = {}
+        signature_version = os.environ.get('BENTOML_S3_SIGNATURE_VERSION', 's3v4')
```

Comment:
Could you add `signature_version` to the BentoML config instead? sorry, we haven't had proper documentation for this yet(you can find some details here https://github.com/bentoml/BentoML/issues/671). 

I think you can add 's3_signature_version' = s3v4 to the `yatai_service` section here:  https://github.com/bentoml/BentoML/blob/master/bentoml/configuration/default_bentoml.cfg

And change this line to `signature_version = config('yatai_service').get('S3_SIGNATURE_VERSION')`

This way user can still set this config via environment variable `BENTOML__YATAI_SERVICE__S3_SIGNATURE_VERSION`

Reply:
The BentoML has  a great configuration system! I updated my PR according to your advice. Thanks for quick feedback.